src/mongo/db/storage/tokuft/tokuft_dictionary.cpp

```diff
@@ -202,6 +231,72 @@ namespace mongo {
         }
     }
 
+    KVDictionary::Cursor *TokuFTDictionary::CreatePrelockedCursorWithRetryAndStartKey(OperationContext *opCtx,
+                                                                                      const Slice &key,
+                                                                                      const int direction) const {
+        const int MAX = 32; // <-- This is a rough guess/maybe maps to threads.
```

Comment:
Regarding the MAX constants added in toku_dictionary.cpp, this should probably be defined once in toku_dictionary.h with a more descriptive name and use it in both methods. In the future we may decide to make it a configurable parameter.